src/frontends/onnx/frontend/src/op/less_or_equal.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
```

Comment:
This declaration should be only in headers, remove please.

Reply:
Oops, forgot to delete that after copy.